### PR TITLE
Future/empty screen

### PR DESCRIPTION
**Pull request by duffleit**
Give a hint if no bills have been added yet

image

### CODE REVIEW

File: src/routes/App/components/EmptyData/EmptyData.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styles from "./EmptyData.scss";
+import withNavigation from "components/hoc/withNavigation";
+import LocationBuilder from "../../modules/navigation/LocationBuilder";
+import SafeBox from "./safebox.png";
+
+export const EmptyData = (props) => {
```
Comment (thomaseizinger) [inline src/routes/App/components/EmptyData/EmptyData.js]: EmptyData sounds too generic IMO. What about "EmptyNobtPlaceholder"?

File: src/routes/App/components/EmptyData/EmptyData.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    height: 30%;
+    max-height: 160px;
+
+    background: no-repeat center;
+    background-size: contain;
+  }
+
+  .topLabel {
+    font-size: 30px;
```
Comment (thomaseizinger) [inline src/routes/App/components/EmptyData/EmptyData.scss]: Remember the react talk from WADC? We should also start creating dedicated components for text that encapsulate styling. Having px-based font-sizes all over the place is hard to be kept consistent.

Comment (duffleit) [inline src/routes/App/components/EmptyData/EmptyData.scss]: You are right, I just don't think that this is part of this feature implementation.

File: src/routes/App/containers/NobtContainer.js [hunk 3]
```diff
@@ -17,7 +17,8 @@ const mapStateToProps = (state) => {
     billSortProperty: getBillSortProperty(state),
     isNobtEmpty: getBills(state).length === 0,
```
Comment (duffleit) [inline src/routes/App/containers/NobtContainer.js]: aaaah, I've overlooked!

File: src/routes/App/components/EmptyData/EmptyData.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.container {
+  font-family: $fancy-font;
+  color: $color-accent;
+  text-align: center;
+
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 40%;
```
Comment (duffleit) [inline src/routes/App/components/EmptyData/EmptyData.scss]: The whole nobt-screen has a serious layout problem IMHO. I can't use flexbox, as I have no possibility to get the height I need (container does not capture 100% and the nesting is inferior). I think you are right, position absolute should be avoided and can be considered as a 	warning signal. BUT, under consideration of the fact, that the whole bill screen will be changed in the near future, we should stay with this tradeoff temporary.

File: src/routes/App/components/EmptyData/EmptyData.scss (hunk 2 above)
Comment (thomaseizinger) [inline src/routes/App/components/EmptyData/EmptyData.scss]: I agree

File: src/routes/App/components/EmptyData/EmptyData.scss (hunk 4 above)
Comment (thomaseizinger) [inline src/routes/App/components/EmptyData/EmptyData.scss]: I agree.

File: src/routes/App/components/EmptyData/EmptyData.scss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+  .icon{
```
Comment (thomaseizinger) [inline src/routes/App/components/EmptyData/EmptyData.scss]: Did you run the auto-formatter on this file?

**Pull request merged**